models/fivetran_platform__connector_status.sql

```diff
@@ -198,12 +198,9 @@ final as (
         connector_recent_logs.last_sync_started_at,
         connector_recent_logs.last_sync_completed_at,
         connector_recent_logs.set_up_at,
-        coalesce(schema_changes.number_of_schema_changes_last_month, 0) as number_of_schema_changes_last_month
-        
-        {% if var('fivetran_platform_using_sync_alert_messages', true) and target.type != 'sqlserver' %}
-        , {{ fivetran_utils.string_agg("distinct case when connector_recent_logs.event_type = 'SEVERE' then connector_recent_logs.message_data else null end", "'\\n'") }} as errors_since_last_completed_sync
-        , {{ fivetran_utils.string_agg("distinct case when connector_recent_logs.event_type = 'WARNING' then connector_recent_logs.message_data else null end", "'\\n'") }} as warnings_since_last_completed_sync
-        {% endif %}
+        coalesce(schema_changes.number_of_schema_changes_last_month, 0) as number_of_schema_changes_last_month,
+        count(distinct case when connector_recent_logs.event_type = 'SEVERE' then connector_recent_logs.message_data else null end) as number_errors_since_last_completed_sync,
```

Comment:
Will `message_data` always be different across each log record for each connector_id? What happens if there's 2 of the same message_data -- is it intended that they will only be counted once as the same error or warning?

Reply:
Good question, in this case we are just using the `message_data` field to count the warning/error entries. The distinct case is not being applied to the message_data, but instead the event_type. So if two message_data of the exact same verbiage are found in the warning event_type, then both will still be counted as a warning since last sync.

Reply:
Ah yes you're right! I was trying to understand why we used distinct previously, but that was to ensure we do not aggregate the same string over and over again. However, like you mentioned we do in fact want to still count those as error warnings with this new field. I will update to remove the distinct clause in the count statement.

Reply:
Thanks for catching this and continuing to dive into this query! Just updated.

Reply:
Gotcha, thanks for digging into why we had it in place before! Looks good!